docs/testing.rst

```diff
@@ -1,8 +1,13 @@
 Testing
 =======
 
-``fiscalyear`` comes with a full test-suite called ``test_fiscalyear``. To run the test-suite, you will need to install the `pytest <https://docs.pytest.org/en/latest/>`_ package.
+``fiscalyear`` comes with a full test-suite called ``test_fiscalyear``. To run the test-suite, you
+will need to install the following packages:
 
+* `pytest <https://docs.pytest.org/en/latest/>`_
+* `pytest-cov <https://docs.pytest.org/en/latest/>`_
+* `pytest-mock <https://docs.pytest.org/en/latest/>`_
+* `pytest-runner <https://docs.pytest.org/en/latest/>`_
```

Comment:
Technically `pytest-cov` and `pytest-runner` aren't required to run the unit tests, only to run them with `setup.py` or with coverage analysis. Not sure if we should leave those out and mention them elsewhere or not.

Reply:
I think that they can be considered dev dependencies and that it is OK to mention them. Making distinctions between mandatory and optional dependencies feels out of scope of this request.